test/runner.js

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env node
+'use strict'; // eslint-disable-line strict
```

Comment:
>Node 6.x should help us :)

Does 6.x allow using block-scope variables without strict mode? That's the issue I was running in to.